### ISSUE TITLE
Escaping `<{` character pair

### ISSUE THREAD

**Issue by adriangrigore** [radare/spp#22]
Is there a way to escape the `<{` character pair? I'm pre-processing a file which has `spp` code inside it as an example.

**Comment by radare** [comment]
no, there’s no way to escape this right now. Patches are welcome :)
